ironfish-cli/src/commands/chain/import.ts

```diff
@@ -87,31 +84,33 @@ export default class ImportSnapshot extends IronfishCommand {
       const bar = CliUx.ux.progress({
         barCompleteChar: '\u2588',
         barIncompleteChar: '\u2591',
-        format: 'Downloading snapshot: [{bar}] {value}% | ETA: {eta}s',
+        format:
+          'Downloading snapshot: [{bar}] {percentage}% | {downloadedSize} / {fileSize} | ETA: {eta}s',
       }) as ProgressBar
 
-      bar.start()
+      bar.start(manifest.file_size, 0, { fileSize })
+      let downloaded = 0
 
       const hasher = crypto.createHash('sha256')
       const writer = snapshotFile.createWriteStream()
+      const finished = promisify(stream.finished)
 
       const response = await axios({
         method: 'GET',
         responseType: 'stream',
         url: `${bucketUrl}/${manifest.file_name}`,
-        onDownloadProgress: (progressEvent: {
-          lengthComputable: number
-          loaded: number
-          total: number
-        }) => {
-          const percentage = Math.floor((progressEvent.loaded / progressEvent.total) * 100)
-          bar.update(percentage)
-        },
       })
 
+      // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-unsafe-call
+      response.data.on('data', (chunk: { length: number }) => {
+        downloaded += chunk.length
+        bar.update(downloaded, { downloadedSize: FileUtils.formatFileSize(downloaded) })
+      })
```

Comment:
We need to listen to error too and handle that from the request, especially since this is a large file.